backend/src/test/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepositoryTest.java

```diff
@@ -1213,6 +1216,51 @@ void deleteOrganization() {
     verify(mockGroup).delete();
   }
 
+  @Test
+  void findUser_notFound_error() {
+    String username = "nonexistent@example.com";
+    UserList mockUserList = mock(UserList.class);
+
+    when(mockUserList.stream()).then(i -> Stream.empty());
+    when(_client.listUsers(isNull(), isNull(), anyString(), isNull(), isNull()))
+        .thenReturn(mockUserList);
+    when(_client.listUsers(anyString(), isNull(), isNull(), isNull(), isNull()))
+        .thenReturn(mockUserList);
+
+    Stream.empty().findFirst().isEmpty();
+    IllegalGraphqlArgumentException caught =
+        assertThrows(IllegalGraphqlArgumentException.class, () -> _repo.findUser(username));
+    assertEquals(
+        "Cannot retrieve Okta user's status with unrecognized username", caught.getMessage());
+  }
+
+  @Test
+  void findUser_success() {
+    String username = "siteadmin@example.com";
+    GroupList mockListGroups = mock(GroupList.class);
+    UserList mockUserList = mock(UserList.class);
+    User mockUser = mock(User.class);
+
+    when(_client.listUsers(
+            isNull(), isNull(), eq("profile.login eq \"" + username + "\""), isNull(), isNull()))
+        .thenReturn(mockUserList);
```

Comment:
With the new okta SDK, this mock will need to be updated since there are two new parameters (both are null). Also they now return `List<User>` rather than the weird `UserList` and `GroupList` objects. If you have any questions about that I am totally free to help whenever!